src/Entity/PackageRepository.php

```diff
@@ -469,6 +469,9 @@ public function getDependents(string $name, int $offset = 0, int $limit = 15, st
         return $this->getEntityManager()->getConnection()->fetchAllAssociative($sql, $args);
```

Comment:
Interesting failure here, it seems to pick up the conditional d.total usage but not the join declaring it? TBH this is quite messy code and I don't expect perfect handling here, but it should at least ignore it rather than error I guess.

Reply:
will investigate in https://github.com/staabm/phpstan-dba/issues/188

Reply:
Cool, no rush, glad to provide this repo as playground :) I have another project with hundreds of pure SQL queries I could run the extension on once it matured a bit, but sadly it's private.